pkts-streams/pom.xml

```diff
@@ -8,7 +8,19 @@
   </parent>
 
   <name>Pkts Streams</name>
-  <groupId>io.pkts</groupId>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
```

Comment:
you don't need this as I upgraded to 17. Also, you don't want to force different versions within the same package. https://github.com/aboutsip/pkts/blob/master/pom.xml#L193